scripts/update-dependencies/Program.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.VersionTools;
+using Microsoft.DotNet.VersionTools.Automation;
+using Microsoft.DotNet.VersionTools.Dependencies;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Microsoft.Dotnet.Scripts
+{
+    public static class Program
+    {
+        private static readonly Config config = Config.Instance;
+
+        public static void Main(string[] args)
+        {
+            Trace.Listeners.Add(new TextWriterTraceListener(Console.Out));
+
+            if (!ParseArgs(args))
+                return;
+
+            CreatePullRequest();
+        }
+
+        private static bool ParseArgs(string[] args)
+        {
+            foreach (string arg in args)
+            {
+                int idx = arg.IndexOf('=');
+                if (idx < 0)
+                {
+                    Console.Error.WriteLine($"Unrecognized argument '{arg}'");
+                    return false;
+                }
+
+                string name = arg.Substring(0, idx);
+                string value = arg.Substring(idx + 1);
+                Environment.SetEnvironmentVariable(name, value);
```

Comment:
Seems kind of messy to use Environment.Set/GetEnvironmentVariable to pass config values around within the app. It seems to me like we could probably use our config builders for this: Microsoft.Extensions.Configuration.{CommandLine,Environment}

Reply:
Why do we do this instead of just setting properties on a `Config` instance?